routing/unified_policies_test.go

```diff
@@ -68,26 +74,49 @@ func TestUnifiedPolicies(t *testing.T) {
 		t.Fatal("expected no policy for amt below min htlc")
 	}
 
-	policy = u.policies[fromNode].getPolicy(550, bandwidthHints)
+	policy = u.policies[fromNode].getPolicy(5500, bandwidthHints)
 	if policy != nil {
 		t.Fatal("expected no policy for amt above max htlc")
 	}
 
-	// For 200 sat, p1 yields the highest fee. Use that policy to forward,
+	// For 200 msat, p1 yields the highest fee. Use that policy to forward,
 	// because it will also match p2 in case p1 does not have enough
 	// balance.
 	policy = u.policies[fromNode].getPolicy(200, bandwidthHints)
 	checkPolicy(
 		policy, p1.FeeBaseMSat, p1.FeeProportionalMillionths,
 		p1.TimeLockDelta,
 	)
+	require.Equal(t, capacity2, policy.capacity)
 
-	// For 400 sat, p2 yields the highest fee. Use that policy to forward,
+	// For 400 msat, p2 yields the highest fee. Use that policy to forward,
 	// because it will also match p1 in case p2 does not have enough
 	// balance. In order to match p1, it needs to have p1's time lock delta.
 	policy = u.policies[fromNode].getPolicy(400, bandwidthHints)
 	checkPolicy(
 		policy, p2.FeeBaseMSat, p2.FeeProportionalMillionths,
 		p1.TimeLockDelta,
 	)
+	require.Equal(t, capacity2, policy.capacity)
+
+	// For 4500 msat we expect to get a reduced capacity if we cannot send
```

Comment:
I think we can improve this test a bit by using `t.Run` here to divide the set of logical assertions in this case, which could very well be individual unit tests in other languages/contexts. 

So we can split it into 5 sub tests here:
  1. 200 msat case 
  2. 400 msat case 
  3. 4500 mst case w/ max htc 
  4. max capacity between two nodes 
  5. no routing info

Reply:
That makes sense, I added a table-driven test in a first refactor commit and then extended it later when capacity is introduced. The refactor commit for the test could also be squashed into the previous renaming one, but I thought this way it's cleaner.